news/2018-09-28-Git-History.adoc

```diff
@@ -0,0 +1,152 @@
+= Using Git for configuration history =
+ehsavoie
+2018-09-28
```

Comment:
Not really a bit deal, but this is the future. Even for the timezone difference :)